contracts/ComptrollerG6.sol

```diff
@@ -0,0 +1,1414 @@
+pragma solidity ^0.5.16;
+
+import "./CToken.sol";
+import "./ErrorReporter.sol";
+import "./PriceOracle.sol";
+import "./ComptrollerInterface.sol";
+import "./ComptrollerStorage.sol";
+import "./Unitroller.sol";
+import "./Governance/Comp.sol";
+
+/**
+ * @title Compound's Comptroller Contract
+ * @author Compound
+ */
+contract ComptrollerG6 is ComptrollerV5Storage, ComptrollerInterface, ComptrollerErrorReporter, ExponentialNoError {
```

Comment:
why a 6 and 7? maybe needs a rebase

Reply:
@coburncoburn - absolutely, this still has the manual claims patch in there!

Reply:
@coburncoburn fixed now.